packages/runtime/id-compressor/src/sessionSpaceNormalizer.ts

```diff
@@ -72,17 +72,18 @@ export class SessionSpaceNormalizer {
 			ranges.push([genCount, count]);
 		}
 
-		if (ranges.length === 0) {
+		const firstEntry = ranges[0];
+		if (ranges.length === 0 || firstEntry === undefined) {
```

Comment:
This isn't equivalent behavior to before. If `firstEntry` is `undefined`, then that's a bug and an error - we shouldn't be returning successfully in that case. Assert not undefined instead.